std/exception.d

```diff
@@ -1300,6 +1304,31 @@ bool mayPointTo(S, T)(auto ref const shared S source, ref const shared T target)
     assert(b.doesPointTo(*aLoc)); // b points to where a is pointing
 }
 
+
+version(unittest)
+{
+    // 17084 : bug doesn't happen if these declaration are local (even static)
+    struct Page17084
```

Comment:
Can those be made `private` or is this what you meant by `local`?

Reply:
they can be made `private`. by 
>local even static

 i meant inside the unittest block. Will make the comment less ambiguous.